TIS-100/33762.0.txt

```diff
@@ -0,0 +1,67 @@
+@0
+S:MOV UP, ACC  # starts a loop with moving the input value to the accumulator
+ZERO:JGZ ONE  # starts "ZERO" loop with checking if the accumulator is greater than 0 (if yes, jumps to "ONE" loop)
+MOV 0, DOWN  # if not greater than 0, moves 0 down
+JMP S  # starts "S" loop again
+ONE:MOV 1, DOWN  # moves 1 down
+MOV UP, ACC  # moves the input value to the accumulator
+JEZ ZERO  # checks if the accumulator is 0 (if yes, jumps to "ZERO" loop)
+MOV 0, DOWN  # moves 0 down 
+JRO -3  # jumps back 3 lines (to moving the input value to the accumulator)
```

Comment:
Why use `JRO` here? Why not jump to a named label (which would be more readable)?